aiida_vasp/workchains/converge.py

```diff
@@ -361,7 +362,7 @@ def _init_pw_context(self):
         self.ctx.converge.pw_iteration = 0
         settings.pwcutoff = None
         try:
```

Comment:
Err the bug was hidden away due to abusing the `try...catch` block. In general, we should avoid using them in the future if possible.